modules/apps/foundation/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/liferay/init.es.js

```diff
@@ -37,7 +37,7 @@ var initSPA = function(callback) {
 					}
 
 					var excluded = Liferay.SPA.excludedPaths.some(
-						(excludedPath) => url.indexOf(excludedPath) === 0
+						(excludedPath) => url.indexOf(themeDisplay.getPathContext() + excludedPath) === 0
```

Comment:
Hi @brunobasto,

I agree with you that users should be able to exclude any path.

Considering that point, in order to avoid a potential breaking change, could we add a replacer/wildcard for the context path at least? I was thinking about a token like _{context-path}_ which can be easily replaced with the current context path value before evaluating the comparison. For example, using it in spa.properties file:

`spa.excluded.paths={context-path}/documents,{context-path}/image`

What do you think? Would it make sense?